src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstiertodeliusupdate/health/QueueHealthTest.kt

```diff
@@ -0,0 +1,177 @@
+package uk.gov.justice.digital.hmpps.hmppstiertodeliusupdate.health
```

Comment:
There's no ingress on this application at the moment, so presumably we would have to be in a shell somewhere nearby curling this to see it? 

If it's standard boilerplate on all our apps then that's fair enough, but I'm not sure what it's adding to this app in particular. The situations we are testing for seem unlikely to arise and the DLQ client is just a dead weight used for reporting only unless we are going to add in the queue management stuff.

Moreover, a lot of this information is available, and more accessible and readable from monitoring with the assoicated alerting.

Reply:
kubernetes uses these endpoints for its own monitoring, to scale down sick pods.

Reply:
OK, so we will need the endpoint. I still feel there's a lot of not very useful information in it - hmpps-tier provides a health endpoint with a couple of lines of code.